client/metrics/metrics.go

```diff
@@ -173,6 +175,16 @@ func initMetrics(constLabels prometheus.Labels) {
 			Help:        "tso batch send latency",
 		})
 
+	TSORetryCount = prometheus.NewHistogram(
+		prometheus.HistogramOpts{
+			Namespace:   "pd_client",
+			Subsystem:   "request",
+			Name:        "tso_retry",
+			ConstLabels: constLabels,
+			Help:        "Counter of retry time for TSO request.",
+			Buckets:     []float64{1, 2, 4, 8, 10, 12, 16, 20},
```

Comment:
```suggestion
			Buckets:     []float64{1, 2, 4, 8, 12, 16, 20},
```

Maybe this is better, otherwise the bucket 8-10, 10-12 is smaller than previous bucket 4-8 ...